.github/workflows/CI.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Continuous Integration Jobs
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        node-version: [16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    name: Unit tests on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # Clone the repository
+      - uses: actions/checkout@v2
+
+      # Set up Node and yarn; use cached dependencies
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "yarn"
+
+      - run: yarn install --frozen-lockfile
+      - run: yarn generate:langfile
```

Comment:
I'm a little confused. why does `generate:langfile` need to be run on PR action?

Reply:
There are unit tests that depend on the existence of `langfile.js`, but the build steps that generate that file are dependent on the configuration created by `yarn setup` (which requires the local installation of Tabletop Playground and therefore isn't really relevant to CI / unit tests). I added `generate:langfile` to skip all of the source copying / symlinking and just generate the langfile.